frontendVfsUtils/src/main/java/com/virtuslab/gitmachete/frontend/vfsutils/GitVfsUtils.java

```diff
@@ -0,0 +1,54 @@
+package com.virtuslab.gitmachete.frontend.vfsutils;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import com.intellij.openapi.vfs.VirtualFile;
+import git4idea.GitUtil;
+import git4idea.repo.GitRepository;
+import io.vavr.control.Option;
+
+public final class GitVfsUtils {
+
+  private static final String MACHETE_FILE_NAME = "machete";
```

Comment:
Maybe make this constant public & use in other contexts (`MacheteFileType`, `OpenMacheteFileAction`) + possibly move to `frontendFile`

Reply:
Yeah, I have been thinking about that and decided to keep these modules independent

Reply:
Are we sure that we want to make `frontendVfsUtils` dependent on `frontendFile` for this constant only? :confused:

Reply:
Right, not worth adding an extra dep, let's leave as is